content/version_control/VCS.tex

```diff
@@ -0,0 +1,174 @@
+  }
+  \item{
+	Sourceforge.net uses this type of versioning.  
+  }
+  \item{
+      \textbf{Example:} SVN (Subversion)
+  }  
+  \end{itemize}
+\column{0.5\textwidth}
+\begin{figure}
+	 \includegraphics[width=.9\textwidth]{images/cvcs}
+	 \caption{Centralized Version System \cite{website}}
+ \end{figure}
+\end{columns}
+\end{frame}	
+
+\begin{frame}{Types of VCS}{Decentralized VCS}
+\subsection{Distributed VCS}
+\begin{columns}
+\column{0.5\textwidth}
+\begin{itemize}
+		  \item {
+		    In distributed VCS, everyone has a local copy of the entire work’s history
+		  }
+		  \item {
+		    Each repository is as good as the other
+		  }
+		  \item{
+			Prone to central server crash
+		  }
+		  \item{
+			Your local copy is a repository, and you can commit to it and get all benefits of source control - Offline Source Control.
+		  }	  
+		  \item{
+			Mozilla Firefox uses this kind of versioning.  
+		  }
+		  \item{
+           \textbf{ Example: }Git		  
+		  }
+	  \end{itemize}
+\column{0.5\textwidth}
+\begin{figure}
+	 \includegraphics[width=.9\textwidth]{images/dvcs}
+	 \caption{Distributed Version System \cite{website}}
+ \end{figure}
+\end{columns}
+\end{frame}	
+
+\begin{frame}[allowframebreaks]{Types of VCS}{Git vs SVN}
+\subsection{Git vs SVN}
+
+\textbf{Pros of using SVN}
+\begin{itemize}
+    \item Subversion has better GUIs than Git
+    \item Working through versions in Git is tougher than SVN. Git uses SHA-1 hashes while SVN uses sequential revision numbers. 
+\end{itemize}
+\textbf{Pros of using Git}
+\begin{itemize}
+    \item Git is much faster than SVN as they do not have to keep communicating with a central server when changes are made.
+    \item Very secure. Every file and commit is checksummed.
+    \item The Git branching model is much better than any other VCS'.
+    \item Git repository file formats are simple and hence  repair and corruption is rare. Space requirements are small
+    \item As it is a DVCS, scalability is not an issue. 
+\end{itemize}
+\framebreak
+\begin{figure}
+	\includegraphics[scale=0.45]{images/better-git}
+	\caption{Comparison of Git and SVN speeds\cite{better}}
+\end{figure}
+
+\end{frame}
+
+
+
+\begin{frame}{Take home message}
+\section{Take home message}
+\begin{itemize}
+    \item There is not much difference between the most popular VCSs. It ultimately comes down to the use case.
+    \item If there is a need for "Offline Source Control" with really good branching features, Git's fantastic. Open source projects are worked on with the help of Git. 
+    \item If there is a need to have a strictly centralized Source Control that is simple and has excellent tooling (at least on Windows), choose SVN.
+\end{itemize}
+    
+\end{frame}
+
+\begin{frame}{References}
+  \section{References}
+    
+  \begin{thebibliography}{10}
+
+  \bibitem{website}
+	\href{https://www.appfusions.com/display/StashSCMImporter/CVCS+vs.+DVCS+In+a+Nutshell
```

Comment:
For some reason when I render the pdf, your links are not showing up. 
Please also commit the pdf, for easier access and review!

Reply:
You mean the references are not clickable?

Reply:
No, the links are not shown in the references slide. They should also be displayed.